File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/operation_history_query_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class OperationHistoryQueryForm:
    action: str = None
    result: str = None
    page: int = None
    size: int = None
    sort_field: str = None
    sort_order: str = None

File: sdkwork-app-sdk-typescript/src/types/operation-history-query-form.ts
export interface OperationHistoryQueryForm {
  /** 操作类型 */
  action?: string;
  /** 操作结果 */
  result?: string;
  /** 页码 */
  page?: number;
  /** 每页大小 */
  size?: number;
  /** 排序字段 */
  sortField?: string;
  /** 排序方向: asc/desc */
  sortOrder?: string;
}
